src/main.js

```diff
@@ -2,6 +2,7 @@ import Vue from 'vue';
 import App from './App.vue';
 import router from './router';
 import store from './store';
+import fetch from 'node-fetch';
```

Comment:
@thetornadotitan This fetch import has to be removed as it is causing a linting error. Is husky being triggered when you commit?